docs/input/docs/learn/branching-strategies/gitflow/examples.md

```diff
@@ -6,61 +6,79 @@ RedirectFrom:
 - docs/git-branching-strategies/gitflow-examples
 ---
 
-These examples are using the _default_ configuration with GitVersion. Which is
-[continuous deployment](/docs/reference/modes/continuous-deployment) mode for
-`develop` and [continuous delivery](/docs/reference/modes/continuous-delivery) mode
-for all other branches.
-
-This default configuration allows you to publish CI builds from develop to a CI
-MyGet feed, or another CI feed. Then all other branches are manually released
-then tagged. Read more about this at [version increments](/docs/reference/version-increments).
+These examples are illustrating the usage of the supported `GitFlow` workflow in GitVersion. 
+To enable this workflow, the build-in template [GitFlow/v1](/docs/workflows/GitFlow/v1.json) needs to be referenced in the configuration as following:
+```yaml
+workflow: GitFlow/v1
+mode: ContinuousDelivery
+```
+
+Where 
+the [continuous deployment](/docs/reference/modes/continuous-deployment) mode for no branches, 
+the [continuous delivery](/docs/reference/modes/continuous-delivery) mode for `main`, `support` and `develop` branches and 
+the [manual deployment](/docs/reference/modes/manual-deployment) mode for `release`, `feature`, `hotfix` and `unknown` branches 
+are specified.
+
+This configuration allows you to publish CI (Continuous Integration) builds from `main`, `support` and `develop` branches to an artifact repository. 
+All other branches are manually published. Read more about this at [version increments](/docs/reference/version-increments).
```

Comment:
I prefer to wrap Markdown content at 80 characters, so the line-length is sensible, making the text easier to read in plain text. I thought that should have been enforced by the configured Markdown linter, but apparently not? 🤔 

```suggestion
This configuration allows you to publish CI (Continuous Integration) builds
from `main`, `support` and `develop` branches to an artifact repository.
All other branches are manually published. Read more about this at [version
increments](/docs/reference/version-increments).
```